packages/maskbook/src/web-workers/OnDemandWorker.ts

```diff
@@ -27,31 +27,77 @@ export class OnDemandWorker extends Worker {
         }, Math.min(this.inactiveTimeToTerminate, WorkerCheckTerminateInterval))
     }
     protected log(...args: any[]) {
-        // console.log(`OnDemandWorker ${this.id}`, ...args)
+        console.log(`OnDemandWorker ${this.init[1]?.name}`, ...args)
```

Comment:
name + unique id

Reply:
It's callsite duty. This options is inherit from the native Worker.